web/src/main/java/se/inera/intyg/webcert/web/service/facade/list/CertificateListItemConverterImpl.java

```diff
@@ -159,7 +159,7 @@ private CertificateListItemStatus getCertificateListItemStatus(String status, Re
             return CertificateListItemStatus.SENT;
         }
 
-        if (status.equals("CANCELLED")) {
+        if (status.equals("CANCELLED") || status.equals("DRAFT_LOCKED_CANCELLED")) {
```

Comment:
Consider changing hard coded string values to private static final Strings. Just so that we dont have checkstyle errors!